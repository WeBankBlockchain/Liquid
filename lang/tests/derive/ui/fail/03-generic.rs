use liquid::InOut;
use liquid_lang as liquid;

#[derive(InOut)]
pub struct MyStruct<T> {
    a: T,
}

fn main() {}
