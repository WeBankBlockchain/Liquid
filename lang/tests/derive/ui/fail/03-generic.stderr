error: generic are not supported
 --> $DIR/03-generic.rs:5:20
  |
5 | pub struct MyStruct<T> {
  |                    ^^^
